docs/self-managed/operational-guides/production-guide.md

```diff
@@ -0,0 +1,572 @@
+---
+id: production-guide
+title: "Helm Chart Production Guide"
+sidebar_label: "Helm Chart Production Guide"
+description: "Learn how to set up the helm chart in a production setting."
+---
+
+## Overview
+
+This guide provides a simplified and streamlined architecture for core Camunda 8 applications using the Camunda Helm Chart. This setup minimizes complexity while offering a reliable foundation for most production use cases.
+
+The goal of this guide is to give you a **scenario-based, production focused, step-by-step guide** for setting up the Camunda Helm Chart. We will use AWS as a reference here but the guide would work with other supported cloud providers and their comparable services as well. By following this guide, you will be familiar with all of the necessary requirements for having a production ready Camunda Helm Chart.
+
+## Prerequisites
+
+Before proceeding with the setup, ensure the following requirements are met:
+
+- **Kubernetes Cluster**: A functioning Kubernetes cluster with kubectl access.
+- **Helm**: Helm CLI installed
```

Comment:
I'm wondering whether it makes sense here to link to somewhere where we state what versions we support or like minimum (> x).
Helm had issues in the past and people will use the guide and expect similar results and could debug differences to their own environments if they run into problems.

Reply:
We have a page where we define versions and keep it up to date. Can we maintain these versions in all places?